vyper/parser/lll_node.py

```diff
@@ -70,6 +70,10 @@ def __init__(self, value, args=None, typ=None, location=None, pos=None, annotati
         if isinstance(self.value, int):
             self.valency = 1
             self.gas = 5
+        elif isinstance(self.value, float):
```

Comment:
are you sure this gets handled correctly? As far as I know the LLL compiler has no concept of `float`. So you will have to remove this. And the multiply the value by the decimal multiplier (in stmt code below) to get an int for consumption by LLL compiler.

Reply:
I double checked this, and you're right; it isn't needed, which means that it isn't being handled properly. To handle this for the decimal type we do need to use the `DECIMAL_DIVISOR` to convert it. Would it make sense to use the function to convert base types in parser_utils.py? Or to handle it in a different manner?